src/cms/templates/feedback/admin_feedback_list.html

```diff
@@ -0,0 +1,121 @@
+{% extends "_base.html" %}
+{% load i18n %}
+{% block content %}
+{% load static %}
+{% load widget_tweaks %}
+<h2>Technical Feedback</h2>
+
+<table class="w-full mt-4 rounded border border-solid border-gray-200 shadow bg-white" border="1">
+    <tr class="border-b border-solid border-gray-200">
+        <th>{% trans 'Type' %}</th>
+        <th>{% trans 'Object' %}</th>
+        <th>{% trans 'Comment' %}</th>
+        <th>{% trans 'Rating' %}</th>
+        <th>{% trans 'Date' %}</th>
+    </tr>
+    {% if not feedback_Admin_exists %} 
```

Comment:
```suggestion
    {% if not feedback_exists %} 
```
The variable passed to the template is called `feedback_exists`, right? Also, if you want to use the name `admin_feedback_exists`, please make to keep your variable names lower case.